src/components/CandidateForm.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+import style from "./CandidateForm.module.css";
+import Button from "./Button";
+
+function CandidateForm(props) {
+  const [items, setItem] = useState(props.inputs);
+
+  const handleChange = (e) => {
+    e.preventDefault();
+    const aux = items;
+    aux.map((item) =>
```

Comment:
.map() function always returns an array. We can store the returned values in a variable and make use of it. In your case you should not assign 'items' to 'aux', you should assign a copy of 'items' to 'aux'. You can do so by mapping over items and return each value to aux (which btw you could call it newItems or smth similar to be more readable).
A cleaner way to change the values from aux would  be:    ```
const aux = items.map(item =>
      item.name !== e.target.name
        ? { ...item }
        : { ...item, value: e.target.value }
    );
```
and then 'setItem()' with the new values